backend/server.js

```diff
@@ -32,7 +32,7 @@ mongoose.connect(process.env.MONGO_URI)
     .then(() => {
         console.log("Connected to DB!")
         app.listen(PORT, () => {
-            console.log(`listening on port ${PORT}!\nhttp://localhost:4000`)
+            console.log(`listening on port ${PORT}!\nhttp://localhost:3000`)
```

Comment:
It is port 4000 not 3000 in package.json there is a proxy that routes it to 3000 for local testing